docs/_reference/auth.md

```diff
@@ -223,6 +223,52 @@ methods:
       }
       ```
 
+  - signature: auth.updatePassword(password)
+    id: updatePassword
+    generator: true
+    description: You can set a user's password.
+    arguments:
+      - name: password
+        required: true
+        type: String
+        description: The user's password.
+    output: A [firebase.Promise](https://firebase.google.com/docs/reference/js/firebase.Promise.html) instance.
```

Comment:
The saga itself doesn't return anything since the promise is resolved and doesn't return anything.

If we disregard the side effects:
```js
return yield call([auth.currentUser, auth.currentUser.updatePassword], password)
// is roughly the same as
return yield Promise.resolve(null)
// is the same as
return null
```

So I think we should remove the `output` key entirely. :)